src/randoop/sequence/AbstractGenerator.java

```diff
@@ -69,7 +67,13 @@
    * Sequence limit for generation. If generation reaches the specified sequence
    * limit, the generator stops generating sequences.
    */
-  public final int maxSequences;
+  public final int maxGeneratedSequences;
+  
+  /**
+   * Limit for output. If more than specified number of sequences are in the
```

Comment:
Should it be equal to rather than more than?  If I specify 500 as the max, I will be surprised to see 501 in the output.